### PR TITLE
Upgrade to aspnet5 beta5 version

### PR DESCRIPTION
**Pull request by xabikos**
Include the required changes to use the latest, beta 5, version of asp.net 5

### CODE REVIEW

File: src/React.AspNet/JsxFileSystem.cs [hunk 1]
```diff
@@ -137,13 +137,13 @@ public IDirectoryContents GetDirectoryContents(string subpath)
...
 		/// </returns>
-		public IExpirationTrigger Watch(string filter)
+		IExpirationTrigger IOwinFileSystem.Watch(string filter)
 		{
 			return _physicalFileSystem.Watch(filter);
 		}
 #endif
 
-		private class JsxFileInfo : IFileInfo
+        private class JsxFileInfo : IFileInfo
```
Comment (Daniel15) [inline src/React.AspNet/JsxFileSystem.cs]: Revert this change please

**Comment by Daniel15** [comment]
This looks pretty good, please use tabs rather than spaces for indentation though. Once you change that, I'll merge this.

**Comment by xabikos** [comment]
Thanks for the quick respond. I made the changes and it should be fine. Any plans for a new release?

**Comment by Daniel15** [comment]
There's still spaces around, but I can fix them separately. I'll try to push a new release soon - In the meantime you can run the dev build (`dev-build.bat`) and use the NuGet packages it builds.
